diff --git niwa-1.0:niwa/nyuukou_demo/fanva-2013.07.15.05.07 HEAD:niwa/nyuukou_demo/fanva-2013.07.16.16.52
index 2358fa9..bdea3ef 100644
--- fanva-2013.07.15.05.07
+++ fanva-2013.07.16.16.52
@@ -1,14 +1,14 @@
-タイトル: 公共性を再演する| 作品の解説を36種類の言語に翻訳する 
-lu za'ure'u gau draci lo ka gubni sei fe lu'e lo ve ciksi be lo larcu cixa bangu
-cu fanva li'u
-
-
-
+タイトル: 公共性を再演する| 作品の解説を36種類の言語に翻訳する
+(丹羽良徳の2004年から2012年の介入プロジェクト)
 
+lu za'ure'u gau draci lo ka gubni
+sei fe lu'e lo ve ciksi be lo larcu cixa bangu cu fanva se'u
 
+to lo poi la .ni,uan.iocinor. cu cmizu'e ke'a ku'o terpla be ca lo nanca be li
+renonovo ga'o bi'o ga'o renopare toi li'u
 
 33
-水たまりAを水たまりBに移しかえる 
+水たまりAを水たまりBに移しかえる
 2012, 2面ビデオインスタレーション (各11min02sec)
 東京・新宿の路上の水たまりを、2011年の東日本大震災の影響で破損した福島第
 一原子力発電所から20km圏内にあり、一時的に警戒区域に指定された福島県双葉郡楢葉町
@@ -138,7 +138,7 @@ mapti lo smuni poi lo tersku cu djica
 
 
 29
-首相官邸前から富士山頂上までデモ行進する 
+首相官邸前から富士山頂上までデモ行進する
 2012, 2面ビデオインスタレーション(各15min25sec)
 毎週金曜日に行われる首相官邸前での原発抗議活動に大きな旗を持参し
 て参加し、その足で富士山の頂上までひとりでデモ行進を続ける。移動するにつれて抗議
@@ -163,7 +163,7 @@ nunjadypo'a
 
 
 28
-首相官邸前から富士山頂上までデモ行進する 
+首相官邸前から富士山頂上までデモ行進する
 2012, 青焼き印刷に油性マーカー (16枚組で各594mm x 841mm)
 同プロジェクトのヴィデオフッテージから抜き出された静止画像を、図面印刷用
 の青焼きにし、さらに油性インクでドローイングを施した。プロジェクトの過程で浮かび
@@ -183,7 +183,7 @@ stali lo po'o valsi
 
 
 27
-首相官邸前から富士山頂上までデモ行進する 
+首相官邸前から富士山頂上までデモ行進する
 2012, 布にインクジェットプリント(1450mm x 2650mm), 木材
 同プロジェクトのヴィデオフッテージから抜き出された静止画像を特大サイズ
 の布にインクジェットプリントした。等身大を越えるサイズで出力することによって画像
@@ -289,7 +289,7 @@ menli tcini
 
 
 23
-36種類の言語に翻訳した作品集を出版する 
+36種類の言語に翻訳した作品集を出版する
 
 2012 プロジェクト
 2004年からのすべての作品プロジェクトを36種類の言語に翻訳した作品集を発行する。本
@@ -320,7 +320,7 @@ cu nitcu cu ca preja kei se danfu .ia le xanri sevzi poi ma'a ponse
 .i to'i sa'a se sarji zoi gy. mirror publishing gy. toi
 
 22
-モスクワのアパートメントでウラジーミル・レーニンを捜す 
+モスクワのアパートメントでウラジーミル・レーニンを捜す
 2012 プロジェクトのドキュメントヴィデオ
 1991年、ソビエト連邦共和国が崩壊して以降もなお、モスクワ市内の家庭に残され
 ている、ソビエトの初代指導者ウラジーミル・レーニンを表すあらゆるイメージを探し出
@@ -351,7 +351,7 @@ gugykruca jikca dincmu toi
 
 21
 イスタンブールで手持ちのお金がなくなるまで、トルコリラとユー
-ロの外貨両替を繰り返す 
+ロの外貨両替を繰り返す
 2011 パフォーマンスのヴィデオドキュメント
 イスタンブールの観光地、グランドバザール周辺の外貨両替所にて、手持ちのお金がなく
 なってしまうまで、トルコリラとユーロの両替を繰り返す。同じような外貨両替所が点在
@@ -376,7 +376,7 @@ noi te kruca lo zdotu'a lo rontu'a .i la'e di'u jarco lo cabna ropno xlajikni
 
 
 20
-ベルンで熊を拍手喝采する 
+ベルンで熊を拍手喝采する
 2011 プロジェクトのドキュメントヴィデオ
 多くの地域住民に協力してもらい、ベルン市で飼育されている熊に対して拍手喝
 采し、ねぎらい、祝福の言葉をかける。このプロジェクトは、ベルン市の象徴的存在であ
@@ -510,7 +510,7 @@ cmecau cu mojgau lo telcku lo slabu prenu nunckini poi nenri lo ti valsi
 
 ni'o paxa mo'o
 
-li terve'u lo mi se ponse bu'u lo tcadu zarci li'u
+lu terve'u lo mi se ponse bu'u lo tcadu zarci li'u
 de'i li renopapa fi'o vreji lo nuntigni skina to lo ci ve vreji toi goi mentu li
 ze pi noxa .e li so pi civo .e li xa pi noxa
 
